server/authUtils.ts

```diff
@@ -0,0 +1,115 @@
+import OpenIdClient from "openid-client";
+import { Request, Response, NextFunction } from "express";
+
+import * as Config from "./config";
+
+type OboToken = {
+  accessToken: string;
+  expiresIn: number;
+};
+
+type CachedOboToken = {
+  token: OboToken;
+  expires: number;
+};
+
+type Scope = string;
```

Comment:
Trenger kanskje ikke egen type for denne?

Reply:
Nei, kan droppe det 👍